sklearn_numba_dpex/kmeans/tests/test_kmeans.py

```diff
@@ -358,27 +362,31 @@ def _get_score_with_centers(centers):
             np.mean(scores_engine_kmeans_plusplus),
         ],
         [
-            -1827.2270605322217,
-            -1027.674264781121,
-            -885.1541877428601,
+            -1827.22702,
+            -1027.674243,
+            -865.257397,
```

Comment:
NB: the engine output changes from the previous version because the `random_state` now go through sklearn random state validation in the `prepare_fit`  step and is transformed as a `np.random.RandomState` object, rather than being passed directly to the driver as an int.